src/index.js

```diff
@@ -0,0 +1,15 @@
+import { AuroHyperlink } from './auro-hyperlink.js';
+
+/**
+ * Register Custom Element.
+ * @param {Object} name - Name to use for custom element.
+ * @returns {void}
+ */
+export function registerComponent(name = 'custom-hyperlink') {
```

Comment:
do we need to change something in the rollup configuration to make sure that this function gets written to `./dist/auro-hyperlink.js`?

Reply:
nevermind, looks like it was due to the `copyfiles` command in `dist:js` not working on windows machines unless the globs are double quoted, I can write a separate issue for this